File: PARouIMPAR.java
package JAVA;
import javax.swing.JOptionPane;
public class PARouIMPAR {
    public static void main(String args[]){

        int numero;

        numero = Integer.parseInt(JOptionPane.showInputDialog("Qual o numero para ser indentificado em Par ou Impar"));

        if (numero % 2 == 0)
        JOptionPane.showMessageDialog(null,"null, esse "+numero+  " e Par" );
        else 
        JOptionPane.showMessageDialog(null,"null, esse "+numero+ " e Impar" );

    }
    
}

File: PARouIMPAR.c
#include <stdio.h>
main() // metodo principal;
{
    int numero;
    printf("\n Qual o numero para se indentificado em PAR ou IMPAR: "); // denição do numero par se escolido;
    scanf("%d",&numero);
     
    if ( numero % 2 == 0)// condição;
    printf("esse numero %d e Par",numero);
    else
     printf ("esse numero %d e Impar",numero);

printf("\n\n<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<Fim de sistema>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");

}